### PR TITLE
[TwigBridge] fix bootstrap_3_layout ChoiceType's expanded label_html

### LINKED ISSUE #41189: [TwigBridge] bootstrap_3_layout ChoiceType's expanded label_html does not work
**Symfony version(s) affected**: 5.1 not sure exactly but its this commit https://github.com/symfony/symfony/blob/239fe04ff9c2ce9c36d7a5ecb103d1db05f25e2e/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig#L102

**Description**
In the nested ternary the `|raw` output is escaped non the less and thus still printed as plain text.

**How to reproduce**
Use bootstrap 3 form theme with ChoiceType expanded & label_html true and some html label.

**Possible Solution**
This has been fixed in the bootstrap 4 template https://github.com/symfony/symfony/blob/5.x/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig#L301-L318 I copied over the solution to a custom template locally and it works (note that bootstrap 3 has the widget in the label unlike in 4+):
```
{% block checkbox_radio_label -%}
    {# Do not display the label if widget is not defined in order to prevent double label rendering #}
    {%- if widget is defined -%}
        {%- if required -%}
            {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' required')|trim}) -%}
        {%- endif -%}
        {%- if parent_label_class is defined -%}
            {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ' ~ parent_label_class)|trim}) -%}
        {%- endif -%}
        {%- if label is not same as(false) and label is empty -%}
            {%- if label_format is not empty -%}
                {%- set label = label_format|replace({
                    '%name%': name,
                    '%id%': id,
                }) -%}
            {%- else -%}
                {% set label = name|humanize %}
            {%- endif -%}
        {%- endif -%}
        <label{% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}>
            {{- widget|raw }}

            <span>
                {%- if label is not same as(false) -%}
                    {%- if translation_domain is same as(false) -%}
                        {%- if label_html is same as(false) -%}
                            {{ label -}}
                        {%- else -%}
                            {{ label|raw -}}
                        {%- endif -%}
                    {%- else -%}
                        {%- if label_html is same as(false) -%}
                            {{ label|trans(label_translation_parameters, translation_domain) -}}
                        {%- else -%}
                            {{ label|trans(label_translation_parameters, translation_domain)|raw -}}
                        {%- endif -%}
                    {%- endif -%}
                {%- endif -%}
            </span>
        </label>
    {%- endif -%}
{%- endblock checkbox_radio_label %}
```

The span is optional, its something personal to help with css selectors as we can not select a text node.

### PR DESCRIPTION
**Pull request by ytilotti**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41189
| License       | MIT

### CODE REVIEW

**Comment by derrabus** [comment]
The bug ticket says that Symfony 5 is affected by this bug. If that were the case, we would need to target the 5.4 branch. Can you please check?

**Comment by ytilotti** [comment]
I can do a PR for 5.4 too. But the bug is available on 6.x

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 1]
```diff
@@ -101,7 +101,22 @@
             {%- endif -%}
         {%- endif -%}
         <label{% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}>
-            {{- widget|raw }} {{ label is not same as(false) ? (translation_domain is same as(false) ? (label_html is same as(false) ? label : label|raw) : (label_html is same as(false) ? label|trans(label_translation_parameters, translation_domain) : label|trans(label_translation_parameters, translation_domain)|raw)) -}}
+            {{- widget|raw }}
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: extra space at end of line
but didn't we have a whitespace here, and it's gone now with the next line starting witn`{-`?
does it matter?

**Review by nicolas-grekas — COMMENTED** [review]
Please remove the issue number from the PR ~description~ title and set it to a meaningful line, this will be useful to readers of the changelog.
About the branch, we merge 5.4 into 6.0 periodically so 6.0 will also get the fix even if we merge this on 5.4

**Comment by ytilotti** [comment]
@nicolas-grekas the issue number of the PR title or description? Title I suppose.

**Comment by derrabus** [comment]
Title. 🙂

**Comment by ytilotti** [comment]
> About the branch, we merge 5.4 into 6.0 periodically so 6.0 will also get the fix even if we merge this on 5.4

Thank you for the information about the cycle.
@nicolas-grekas, In my case I created the PR on 6.0. I can't change the original branch (maybe a rebase?). Either I recreate the PR or? It's not possible to change the from of PR.

Sorry for the inconvenience but it will be taken care of for the next ones.

**Comment by derrabus** [comment]
```sh
git rebase 6.0 --onto 5.4
```

If rebase aborts because of conflicts, solve them and run:
```sh
git add .
git rebase --continue
```

Once the rebase completed, run:
```sh
git push -f
```

Afterwards, you should be able to switch the base branch to 5.4.

**Comment by ytilotti** [comment]
Thanks @derrabus, I was close with my rebase ;) I must have missed a parameter!

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig (hunk 1 above)
Comment (ytilotti) [reply src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: I delete the extra space. It's not necessary.

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @ytilotti.

**Pull request merged**